yaml/pois.yaml

```diff
@@ -172,6 +172,14 @@ filters:
     min_zoom: 17
     output:
       kind: {col: tags->zoo}
+  - filter: {waterway: waterfall}
+    min_zoom: CASE WHEN to_number(tags->'height', '9999.99') < 10 THEN 15
```

Comment:
I've incorporated your stuff in https://github.com/mapzen/vector-datasource/commit/6924d7a4236ecd347d87a02ed4ee2f0ea376eaed into this branch via 366a79c4ab316807ebfae1cc9a68eaf9810bf495 and 5d55d4e525fcd45db4b1df884d425c65ce1027f9. It's also rebased against master now.